docs/release-notes/8.7.asciidoc

```diff
@@ -0,0 +1,83 @@
+* Improves the UI for building an Investigation Guide query ({pull}150363[#150363]).
+* Adds the ability to create a rule from a Timeline ({pull}143020[#143020]).
+
+[discrete]
+[[bug-fixes-8.7.0]]
+==== Bug fixes and enhancements
+* Various bug fixes and UX enhancements for the Alerts page ({pull}152402[#152402]).
+* Improves the formatting and readability of machine learning job names ({pull}148974[#148974], {pull}148780[#148780]).
+* Introduces cross-cluster search support for Indicator Match rules by improving rule performance ({pull}149113[#149113]).
+* Warns you about type conflicts and unmapped indices when you're creating rule exceptions ({pull}149149[#149149]).
+* Improves sorting of the Rules table: allows you to sort it by any column, removes the *Advanced sorting* toggle, and removes the unnecessary `Version` column ({pull}149840[#149840]).
+* Adds a *Clear table filters* button to the Rules page so you can clear all filters in one click ({pull}150059[#150059]).
+* Adds buttons to the Rules page that allow you to view only enabled or disabled rules ({pull}150153[#150153]).
+* The *Related alerts by process ancestry* section of the Alert details flyout is now GA ({pull}152011[#152011]).
+* Adds the option to suppress custom query rule alerts during a specific time window. Duplicate alerts within that time window will be grouped ({pull}148868[#148868]).
+* Reduces alert creation errors by stopping the detection engine from writing non-ECS-compliant fields to alerts from source events ({pull}147628[#147628]).
+* Simplifies the interface for navigating from a rule's details page back to the Rules page ({pull}147357[#147357]).
+* Allows you to resize the Rule preview panel ({pull}147351[#147351]).
+* Improves the Bulk Edit API by adding a `skipped` property to rules that weren't updated, and updates the toast message to show which were skipped ({pull}147345[#147345]).
+* Allows placeholder fields in Osquery queries ({pull}146598[#146598]).
+* Allows the Rules table state to persist even after you refresh or navigate to another page ({pull}145111[#145111]).
+* Improves data fetch performance throughout {elastic-sec}, especially for deployments with large indices and many integrations ({pull}142904[#142904]).
+* Fixes a bug that could cause your cursor to jump to the end of the text field when editing a rule action message ({pull}150823[#150823]).
+* Fixes a bug that could result in incorrect links to machine learning jobs from search results ({pull}150881[#150881]).
+* Fixes a bug that caused a fade in and out effect on alert descriptions ({pull}150998[#150998]).
+* Fixes a bug that caused the Alerts page to default to the wrong chart type ({pull}151073[#151073]).
+* Fixes a bug that could hide some shared exception lists when you changed the number of rows in the exceptions lists view ({pull}151393[#151393]).
+* Removes a blank option from the *Field* browser in the Add rule exception flyout ({pull}151398[#151398]).
```

Comment:
Can we put all the "Add rule exception flyout" bugs together one after the other? (Lines 69, 72, and 74).